src/main/java/com/brandwatch/internship/fetchandstoretweets/services/MentionService.java

```diff
@@ -0,0 +1,49 @@
+package com.brandwatch.internship.fetchandstoretweets.services;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Mention;
+import com.brandwatch.internship.fetchandstoretweets.entities.Query;
+import com.brandwatch.internship.fetchandstoretweets.repositories.MentionsRepository;
+import com.brandwatch.internship.fetchandstoretweets.repositories.QueryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.social.twitter.api.Tweet;
+import org.springframework.social.twitter.api.Twitter;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Singleton;
+import java.util.List;
+
+@Service
+@Singleton
```

Comment:
Singleton should be the default scope of `@Service`, so I don't think you need this